lib/extensions/conditional/active_record/base.rb

```diff
@@ -32,6 +32,20 @@ def specific_conditions
     def conditions_satisfied_by?(course_user)
       conditions.all? { |condition| condition.satisfied_by?(course_user) }
     end
+
+    # Permit the conditional for the given course user.
+    #
+    # @param [CourseUser] _course_user The course user in which the conditional is to unlock for
+    def permitted_for!(_course_user)
+      raise NotImplementedError
```

Comment:
@custode you should provide an exception message stating why the error is raised.